server/utils/assessmentUtils.ts

```diff
@@ -28,6 +28,8 @@ export const formatAssessmentData = (
       const linkedtoRoSH = crimNeeds[areaConfig.crimNeedsKey][`${areaConfig.crimNeedsSubKey}LinkedToHarm`] === 'YES'
       const linkedtoReoffending =
         crimNeeds[areaConfig.crimNeedsKey][`${areaConfig.crimNeedsSubKey}LinkedToReoffending`] === 'YES'
+      const linkedtoStrengthsOrProtectiveFactors =
```

Comment:
given that we also have the YesNo field below, would it make sense to rename this (and the two above it) to `isLinkedToStrengthsOrProtectiveFactors` to try and make it a bit clearer?

Reply:
e.g. I'm slightly surprised that something like this is not the default for eslint https://typescript-eslint.io/rules/naming-convention/#enforce-that-boolean-variables-are-prefixed-with-an-allowed-verb